### PR TITLE
Migrate to latest tech docs

### PR DESCRIPTION
**Pull request by nickcolley**
This pull requests brings the project up-to-date with the latest Design System by updating the base tech docs gem and updating the examples to use the new patterns instead of GOV.UK Elements.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.3
```
Comment (nickcolley) [inline .ruby-version]: add a new eol

File: source/_example_snippets/form_fields_no_label/_bad.erb [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+    <div class="govuk-radios govuk-radios--inline" <% if right_align_inputs %>style="float: right;"<% end %>>
+      <div class="govuk-radios__item">
+        <input class="govuk-radios__input" id="wrong-id" name="humantest" type="radio" value="yes">
+        <label class="govuk-label govuk-radios__label" for="humantest-yes">
+          Yes
+        </label>
+      </div>
+      <div class="govuk-radios__item">
+        <input class="govuk-radios__input" id="wrong-id" name="humantest" type="radio" value="no">
+        <label class="govuk-label govuk-radios__label" for="humantest-no">
```
Comment (nickcolley) [inline source/_example_snippets/form_fields_no_label/_bad.erb]: wrong-id is in the wrong place

File: source/_example_snippets/form_fields_no_label/_bad.erb [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
+  <input class="govuk-input govuk-!-width-one-quarter" id="year" name="year" type="text" autocomplete="off" <% if right_align_inputs %>style="float: right;"<% end %>/>
 </div>
 
-<div class="form-group">
-  <fieldset>
-
-    <div class="form-label-bold" <% if right_align_inputs %>style="float: left;"<% end %>>
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend" <% if right_align_inputs %>style="float: left;"<% end %>>
```
Comment (nickcolley) [inline source/_example_snippets/form_fields_no_label/_bad.erb]: legend should be div in bad

**Comment by selfthinker** [comment]
Thanks a lot for doing this update.

I have found two smaller issues which I have just fixed.
(Would you like to have a quick look?)

Another thing I found is that the header is more wonky now when resizing and the left-hand side of it is not lining up with the left-hand side of the content. But I have verified that that is part of the Tech Docs as well, so nothing in this PR. I'll open an issue in the Tech Docs for that (if there isn't one already).

**Pull request merged**